compiler/fir/raw-fir/light-tree2fir/src/org/jetbrains/kotlin/fir/lightTree/converter/DeclarationsConverter.kt

```diff
@@ -958,6 +950,18 @@ class DeclarationsConverter(
                                 ?: FirDefaultPropertySetter(null, session, FirDeclarationOrigin.Source, returnType, propertyVisibility)
                         } else null
 
+                    status = FirDeclarationStatusImpl(propertyVisibility, modifiers.getModality()).apply {
+                        isExpect = modifiers.hasExpect()
+                        isActual = modifiers.hasActual()
+                        isOverride = modifiers.hasOverride()
+                        isConst = modifiers.isConst()
+                        isLateInit = modifiers.hasLateinit()
+                        isInline = modifiers.hasInline() ||
+                                (!isVar && getter!!.isInline) || (isVar && getter!!.isInline && setter!!.isInline)
```

Comment:
IMO this is over-complicated. Something like `getter!!.isInline && setter?.isInline == true` looks much simpler. Same thing for code below.

Reply:
Ditto.